packages/suite/src/components/suite/banners/SuiteBanners/SuiteBanners.tsx

```diff
@@ -53,6 +55,15 @@ export const SuiteBanners = () => {
     let priority = 0;
     if (device?.id && firmwareHashInvalid.includes(device.id)) {
         banner = <FirmwareHashMismatch />;
+        priority = 92;
+    } else if (
+        !isFirmwareRevisionCheckDisabled &&
+        device?.features &&
```

Comment:
Intentionally not checking the `!online` here. Downloading the `releases.json` may fail (blocked, filtered, ..) regardless of being online. 

For example, when testing it and filtering it in the networks tab :)